src/main/resources/filter/web/positioning.filter

```diff
@@ -1,21 +1,27 @@
 # Positioning filter file for recheck.
 
-# Ignore positioning CSS attributes:
+# Ignore attributes that are concerned with positioning:
 
+attribute=top
 attribute=bottom
-attribute=clip
+attribute=right
 attribute=left
+
+attribute=clip
+
 attribute=margin-top
 attribute=margin-right
 attribute=margin-bottom
 attribute=margin-left
+
 attribute=object-position
 attribute=outline
+attribute=absolute-outline
```

Comment:
If your fixes were successful, `absolute-outline` should never pop up.

Reply:
That's not true. We changed that...